### PR TITLE
Refactor TestFields unittest to use Panama data types

### PR DESCRIPTION
**Pull request by mairooni**
#### Description

This PR updates the `TestFields` unittest to use the latest Panama types that TornadoVM provides. 
#### Backend/s tested

Mark the backends affected by this PR.

- [x] OpenCL
- [x] PTX
- [x] SPIRV

#### OS tested

Mark the OS where this PR is tested.

- [x] Linux
- [ ] OSx
- [ ] Windows

#### Did you check on FPGAs?

If it is applicable, check your changes on FPGAs.

- [ ] Yes
- [x] No

#### How to test the new patch?

Run `tornado-test -V uk.ac.manchester.tornado.unittests.fields.TestFields`

### CODE REVIEW

**Review by jjfumero — APPROVED** [review]

File: tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fields/TestFields.java [hunk 1]
```diff
@@ -201,41 +201,33 @@ public void testSetNestedArray() {
...
         Foo(int elements) {
-            //            output = new IntArray(elements);
-            //            a = new IntArray(elements);
-            //            b = new IntArray(elements);
-            output = new int[elements];
-            a = new int[elements];
-            b = new int[elements];
+                        output = new IntArray(elements);
+                        a = new IntArray(elements);
+                        b = new IntArray(elements);
```
Comment (stratika) [inline tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fields/TestFields.java:211]: can you please check the formatter? the spacing seems odd.

Comment (mairooni) [reply tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fields/TestFields.java:211]: I ran `make checkstyle` before committing and it passed

Comment (stratika) [reply tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fields/TestFields.java:211]: My understanding is that `checkstyle` fails, I opened a new PR to fix it. 

I used the `scripts/templates/eclipse-settings/Tornadovm_eclipse_formatter.xml` from the `Adapter from Eclipse Code Formatter` in IntelliJ and it changed those lines.

Comment (mairooni) [reply tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fields/TestFields.java]: Hm....I ran it again now and I am getting the errors you mentioned in the PR #415. I don't get any errors regarding this unittest, but I just noticed that we have the checkstyle off for all unittests. I removed the extra white space.

Comment (stratika) [reply tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fields/TestFields.java]: thanks @mairooni.

**Review by stratika — APPROVED** [review]

**Pull request merged**